corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -557,8 +557,8 @@ hqDefine("cloudcare/js/formplayer/app", function () {
         var progressView = FormplayerFrontend.regions.getRegion('loadingProgress').currentView,
             progressFinishTimeout = 200;
 
-        if (progressView) {
-            progressView.setProgress(1, progressFinishTimeout);
+        if (progressView && progressView.hasProgress()) {
+            progressView.setProgress(1, 1, progressFinishTimeout);
```

Comment:
I don't think this was working before. The intent is to rapidly fill up the progress bar before hiding the screen, but this was causing the progress to **drop** to 1 and show a brief "Completed: 1/200" message, which was weird.